internal/testrunner/runners/system/runner.go

```diff
@@ -1141,6 +1184,18 @@ func (r *runner) validateTestScenario(ctx context.Context, result *testrunner.Re
 		return result.WithError(err)
 	}
 
+	// TODO: remove this once Elasticsearch will map event.original correctly (8.14)
```

Comment:
This is https://github.com/elastic/elasticsearch/pull/106714 I assume? Maybe worth to add the link here.